### PR TITLE
update weights (sync with v0.9.29)

### PR DESCRIPTION
**Pull request by coderobe**
https://github.com/paritytech/polkadot/pull/5988 on master

### CODE REVIEW

**Review by ggwpez — APPROVED** [review]
Looks sane. Apparently the Weights v1.5 update did not format the templates, so there is a fmt needed.  

[SWC](https://weights.tasty.limo/compare?repo=polkadot&threshold=10&path_pattern=runtime%2F**%2Fsrc%2Fweights%2F**%2F*.rs&method=guess-worst&ignore_errors=true&unit=time&old=release-v0.9.29&new=coderobe%2Fweights-29&git_pull=false) and PDF for posterity: [SWC-Web.pdf](https://github.com/paritytech/polkadot/files/9534906/SWC-Web.pdf)

File: runtime/kusama/constants/src/weights/block_weights.rs [hunk 1]
```diff
@@ -42,23 +42,24 @@ use frame_support::{
...
 	///
 	/// Percentiles nanoseconds:
-	///   99th: 6_332_047
-	///   95th: 6_308_225
-	///   75th: 6_236_204
-	pub const BlockExecutionWeight: Weight = WEIGHT_PER_NANOS.saturating_mul(6_192_341);
+	///   99th: 390_162
+	///   95th: 383_320
+	///   75th: 356_698
+	pub const BlockExecutionWeight: Weight = WEIGHT_PER_NANOS.saturating_mul(356_760);
```
Comment (ggwpez) [inline runtime/kusama/constants/src/weights/block_weights.rs:57]: I will double check this on bm2, sees weird.

**Review by chevdor — APPROVED** [review]

File: runtime/kusama/constants/src/weights/block_weights.rs (hunk 1 above)
Comment (coderobe) [reply runtime/kusama/constants/src/weights/block_weights.rs]: reverted

**Comment by coderobe** [comment]
bot merge

**Pull request merged**